app/sidekiq/form1010cg/submission_job.rb

```diff
@@ -51,14 +49,21 @@ def perform(claim_id)
       end
     rescue CARMA::Client::MuleSoftClient::RecordParseError
       StatsD.increment("#{STATSD_KEY_PREFIX}record_parse_error", tags: ["claim_id:#{claim_id}"])
+      StatsD.increment('silent_failure_avoided_no_confirmation', tags: DD_ZSF_TAGS)
+
+      self.class.attempt_send_failure_email(claim)
     rescue => e
       log_exception_to_sentry(e)
       StatsD.increment("#{STATSD_KEY_PREFIX}retries")
 
       raise
     end
 
-    def self.send_failure_email(parsed_form)
+    def self.attempt_send_failure_email(claim)
```

Comment:
Renamed this since there is a chance it will not actually send it.

Reply:
Yeah, all network calls are an attempt. I was mostly trying to better communicate that if the Flipper toggle is off or if there is no email in the parsed form, the email will not even try to send. I agree I like the succinctness of just `send_failure_email` too though.

Reply:
After thinking about it a bit more, I think I agree with you that `send_failure_email` is the better name here. I'll updated it!